datamap/bloom/pom.xml

```diff
@@ -11,7 +11,7 @@
   </parent>
 
   <artifactId>carbondata-bloom</artifactId>
-  <name>Apache CarbonData :: Bloom Index DataMap</name>
+  <name>Apache CarbonData :: Bloom DataMap</name>
```

Comment:
Do not modify this. If you want to move this module to another folder, please raise another PR

Reply:
And you can move secondary index into 'index' folder

Reply:
done